composer.json

```
@@ -34,22 +34,22 @@
         "ext-apcu": "*",
         "ext-json": "*",
         "bbc-rmp/cloudwatch-monitoringhandler": "^1.0",
-        "bbc-rmp/translate": "^1.8.9",
+	"symfony/translation": "4.3.2",
```

```suggestion
	"symfony/translation": "4.3.2",
```
THIS IS UNACCEPTABLE!! not enough spaces :-P